tests/test_cli.py

```diff
@@ -119,3 +119,48 @@ def test_cli_wrong_definitions_name():
         ],
     )
     assert result_valid.exit_code == 1
+
+
+def test_cli_custom_dimensions():
+    """Check that CLI runs through with a non-default dimension"""
+
+    result_valid = runner.invoke(
+        cli,
+        [
+            "validate-project",
+            str(TEST_DATA_DIR / "non-default_dimensions_passing"),
+            "--dimensions",
+            "['variable', 'region', 'foo']",
+        ],
+    )
+    assert result_valid.exit_code == 0
+
+
+def test_cli_custom_dimensions_empty():
+    """Check that CLI raises an expected error with an empty non-default dimension"""
```

Comment:
This could be a bit more clear, something like:
```suggestion
    """Check that CLI raises an error when specifying a non-existent directory ('foo')"""
```